skiko/src/androidMain/kotlin/org/jetbrains/skiko/Renderer.android.kt

```diff
@@ -29,6 +47,276 @@ class SkikoSurfaceView(context: Context, layer: SkiaLayer) : GLSurfaceView(conte
     fun scheduleFrame() {
         frameDispatcher.scheduleFrame()
     }
+
+    private val simpleGestures = GestureDetector(context, object: SimpleOnGestureListener() {
```

Comment:
Not sure if this whole logic belongs to Renderer.android.kt, better move to separate file.

Reply:
Done.